cppcache/include/geode/CacheStatistics.hpp

```diff
@@ -97,11 +97,11 @@ class APACHE_GEODE_EXPORT CacheStatistics {
   virtual time_point getLastAccessedTime() const;
 
  private:
-  virtual void setLastAccessedTime(time_point lat);
-  virtual void setLastModifiedTime(time_point lmt);
+  virtual void setLastAccessedTime(const time_point& tp);
```

Comment:
A time_point boils down to maybe a `long long`, is there any pressing reason to pass by reference, and thus cascade a const reference type change to this interface through all the derived implementations in all client code?

Reply:
And also, this might alter ABI signature. Something I had not into account by the time of preparing this PR. Thanks!